protocol-units/bridge/move-modules/sources/atomic_bridge_counterparty.move

```diff
@@ -159,6 +159,23 @@ module atomic_bridge::atomic_bridge_counterparty {
             },
         );
     }
+
+    public entry fun noop_for_testing(_signer: &signer) {
+         
+    }
+
+    public entry fun pass_data_for_testing(
+        _signer: &signer,
+        originator: vector<u8>,
+        bridge_transfer_id: vector<u8>,
+        hash_lock: vector<u8>,
+        time_lock: u64,
+        recipient: address,
+        amount: u64,
+
+    ) {
+         
+    }
```

Comment:
We want to remove both
`public entry fun noop_for_testing(_signer: &signer)`
And
`public entry fun pass_data_for_testing`